app/controllers/adopter_applications/chats_controller.rb

```diff
@@ -0,0 +1,6 @@
+class AdopterApplications::ChatsController < ApplicationController
+  def show
+    @match = AdopterApplication.find(params[:adopter_application_id])
```

Comment:
Why are we using `AdopterApplication` instead of `Adoption` here? Seems intuitive to use `Adoption`.

Reply:
Good question, I think `Adoption` makes more sense as you said. I don't have any strong ties to the choice to use AdopterApplication. Whatever we use, it just needs to be related to the Adopter & Staff.